### PR TITLE
feature: add disableDragToFlip prop to disable drag flip when needed

### PR DESCRIPTION
**Pull request by siderisng**
Hi there, I added a prop to disable dragging and flipping page when needed, let me know if that looks good to you :)

### CODE REVIEW

File: src/Flipbook.vue [hunk 1]
```diff
@@ -177,6 +179,9 @@ export default
     clickToZoom:
       type: Boolean
       default: true
+    disableDragToFlip:
+      type: Boolean
+      default: false
```
Comment (ts1) [inline src/Flipbook.vue:184]: I prefer positive name (dragToFlip) and defaults to true.
What do you think?

File: src/Flipbook.vue [hunk 2]
```diff
@@ -259,7 +264,7 @@ export default
         'zoom-in'
       else if @clickToZoom and @canZoomOut
         'zoom-out'
-      else
+      else if not @disableDragToFlip
         'grab'
```
Comment (ts1) [inline src/Flipbook.vue:268]: Don't we need `else`?  (I think `auto`)

File: src/Flipbook.vue [hunk 3]
```diff
@@ -697,14 +702,15 @@ export default
       @touchStartX = touch.pageX
       @touchStartY = touch.pageY
       @maxMove = 0
-      if @zoom <= 1
+      if @zoom <= 1 && not @disableDragToFlip
```
Comment (ts1) [inline src/Flipbook.vue:705]: I prefer `and` over `&&`

**Review by ts1 — COMMENTED** [review]
Thank you verfy much for the contribution!
I added some comments so could you check them?

**Comment by siderisng** [comment]
thanks for the suggestions, I thought they were all valid and implemented them. And thanks for creating this library, it saved me a lot of time!

**Comment by ts1** [comment]
Thanks, merged.

**Pull request merged**